lib/logger-utils.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
```

Comment:
We should delete this file

Reply:
Is it okay to take it in a separate PR where we do the complete refactoring of logger logic ?

Reply:
This `logger-utils.js` file is unnecessary. Just delete it.